yarn.lock

```diff
@@ -2624,6 +2629,26 @@ cacache@^10.0.4:
     unique-filename "^1.1.0"
     y18n "^4.0.0"
 
+cacache@^11.2.0:
```

Comment:
@marcdumais-work could you run license check please?

Reply:
For you Anton, sure :) 
I'll report back.

Reply:
I ran the license check and everything looks good